packages/core/admin/ee/server/src/controllers/admin.ts

```diff
@@ -66,4 +70,128 @@ export default {
 
     return { data };
   },
+
+  async getAiUsage(ctx: Context) {
+    const ERROR_PREFIX = 'AI usage data request failed:';
+    const USER_ERROR_MESSAGE = 'AI usage data request failed. Check server logs for details.';
+    // Security check: Ensure user is authenticated and has proper permissions
+    if (!ctx.state.user) {
+      return ctx.unauthorized('Authentication required');
+    }
+
+    // Check if EE features are enabled first
+    if (!strapi.ee?.isEE) {
+      strapi.log.error(`${ERROR_PREFIX} Enterprise Edition features are not enabled`);
+      return ctx.internalServerError(USER_ERROR_MESSAGE);
+    }
+
+    // Get the EE license
+    // First try environment variable, then try reading from file
+    let eeLicense = process.env.STRAPI_LICENSE;
+
+    if (!eeLicense) {
+      try {
+        const licensePath = path.join(strapi.dirs.app.root, 'license.txt');
+        eeLicense = fs.readFileSync(licensePath).toString();
+      } catch (error) {
+        // License file doesn't exist or can't be read
+      }
+    }
+
+    if (!eeLicense) {
+      strapi.log.error(
+        `${ERROR_PREFIX} No EE license found. Please ensure STRAPI_LICENSE environment variable is set or license.txt file exists.`
+      );
+      return ctx.internalServerError(USER_ERROR_MESSAGE);
+    }
+
+    const aiServerUrl = process.env.STRAPI_ADMIN_AI_URL || process.env.STRAPI_AI_URL;
```

Comment:
Minor thing, how come we maintain 2 env vars still?

Reply:
Yeah, we should only have one (even though it's internaluse only, still confusing for us 😆)